documentation/installation.md

```diff
@@ -6,36 +6,35 @@ This document contains the steps for installing and configuring Red Hat OpenShif
 
 ### OpenShift Cluster
 
-Prior to deploying OpenShift AI, it is recommended you review the [Supported Configurations](https://access.redhat.com/articles/rhoai-supported-configs) documentation.
+- (Recommended) Review [Supported Configurations](https://access.redhat.com/articles/rhoai-supported-configs) documentation.
 
-Ensure that you have cluster-admin access to an OpenShift cluster, since we will be installing several operators and configuring various components on the cluster.
+- Account with appropriate permissions for installing operator and configuration installation (typically `kubeadmin`).
 
-The cluster must also have a functional storage provisioner available with a default StorageClass.
+- Functional storage provisioner available with a default StorageClass.
 
-For GPU deployments, this repo is designed specifically to work with AWS to provision additional GPU nodes, but this can still act as an example to deploy GPU resources in any cloud environment or a self-hosted cluster with some minor modifications.
+> [!Note]  
+> If using GPUs (not required): This repo is designed and tested to work with AWS for provisioning additional GPU nodes. 
+> 
+> Can still act as an example to deploy GPU resources in any cloud environment or a self-hosted cluster with some minor modifications.
 
-> [!NOTE]  
+> [!Tip]  
 > Red Hat employees can request a demo cluster using [demo.redhat.com](https://demo.redhat.com) to provision OpenShift AI. For more information see the [Red Hat Demo Environment](redhat_demo_environment.md) documentation.
 
 ### Client Tooling
 
-The bootstrap script relies on the following command line tools. If they're not already available on your system path, the bootstrap script will attempt to download them from the internet, and will place then in a `.\tmp` folder location where the bootstrap script was run:
+The following are required for the bootstrap scripts. If unavailable the scripts will attempt to download required tools and store them at `.\tmp`:
 
-- [oc](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html) - the OpenShift command-line interface (CLI) that allows for creation of applications, and can manage OpenShift Container Platform projects from a terminal.
+- [oc](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html) - OpenShift command-line interface (CLI).
```

Comment:
For the OpenShift docs we can replace a specific version tag (e.g. `4.11`) with the word `latest` to get whatever the newest version is, for example:

https://docs.openshift.com/container-platform/**latest**/cli_reference/openshift_cli/getting-started-cli.html